tests/main/views/test_outcome.py

```diff
@@ -150,10 +150,11 @@ def test_award_brief_post_raises_400_if_required_fields_not_filled(self):
         assert res.status_code == 400
 
         validation_message = document.xpath('//span[@class="validation-message"]')[0].text_content()
-        assert validation_message.strip() == "You need to answer this question."
+        assert validation_message.strip() == "Select a supplier."
 
         error_span = document.xpath('//span[@id="error-brief_response"]')[0]
-        assert self._strip_whitespace(error_span.text_content()) == "Youneedtoanswerthisquestion."
+        # assert that a framework iteration specific message has been rendered
+        assert len(self._strip_whitespace(error_span.text_content())) > 0
```

Comment:
If we don't care about the content of the message, do we need the `strip_whitespace`? Can we just assert that exactly 1 error message has been shown?